modules/op-release-notes-1-17.adoc

```diff
@@ -0,0 +1,168 @@
+// This module is included in the following assemblies:
+// * release_notes/op-release-notes-1-16.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="op-release-notes-1-17_{context}"]
+= Release notes for {pipelines-title} General Availability 1.17
+
+With this update, {pipelines-title} General Availability (GA) 1.17 is available on {OCP} 4.15 and later versions.
+
+[id="new-features-1-17_{context}"]
+== New features
+
+In addition to fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.17:
+
+[id="pipelines-new-features-1-17_{context}"]
+=== Pipelines
+
+* With this release, you can configure multiple Git providers, or you can add multiple configurations for the same Git provider, in the `TektonConfig` custom resource (CR). You can then specify a Git configuration to use in different task runs and pipeline runs by using the `configKey` parameter.
```

Comment:
I am not sure how to phrase that but maybe we need to clarify this is a feature for tekton resolvers and not about Pipelines-as-Code

Reply:
Good point, I am changing it to `With this release, you can configure multiple Git providers by using the Git resolver,...` I think that would work better, WDYT?